renting/setting-up-renterd/linux.md

```diff
@@ -24,167 +24,208 @@ This guide will walk you through setting up `renterd` on Linux. At the end of th
 
-## Setting up a systemd service
+---
+
+## Setting up a system user
 
-Now that you have a recovery phrase, we will create a new system user and `systemd` service to securely run `renterd` on startup.
+Now that you have a recovery phrase, we will create a new system user and `systemd` service to run `renterd` securely on startup.
 
 First, we will create a new system user with `useradd` and disable the creation of a home directory. This is a security precaution that will isolate `renterd` from any unauthorized access to our system. We will then use `usermod` to lock the account and prevent anyone from logging in under the account.
 
+{% tabs %}
+{% tab title="Main Net" %}
 ```console
 sudo useradd -M renterd &&\
 sudo usermod -L renterd
 ```
+{% endtab %}
+
+{% tab title="Zen Test Net" %}
+```console
+sudo useradd -M renterd_zen &&\
+sudo usermod -L renterd_zen
+```
+{% endtab %}
+{% endtabs %}
 
 Now, we will create a new folder under `/var/lib/` titled `renterd` and give it the appropriate permissions. This folder will be utilized specifically to store data related to the `renterd` software. Open the Terminal Emulator and run the following commands:
 
+{% tabs %}
+{% tab title="Main Net" %}
 ```console
 sudo mkdir /var/lib/renterd &&\
 sudo chown renterd:renterd /var/lib/renterd &&\
 sudo chmod o-rwx /var/lib/renterd
 ```
+{% endtab %}
+
+{% tab title="Zen Test Net" %}
+```console
+sudo mkdir /var/lib/renterd_zen &&\
+sudo chown renterd_zen:renterd_zen /var/lib/renterd_zen &&\
+sudo chmod o-rwx /var/lib/renterd_zen
+```
+{% endtab %}
+{% endtabs %}
+
+---
 
-Next, create a file name `renterd.yml` file under `/var/lib/renterd/`
+## Configure your `renterd.yml` file
 
+To begin, create a file name `renterd.yml` file under `/var/lib/renterd/`
+
+{% tabs %}
+{% tab title="Main Net" %}
 ```console
 sudo nano /var/lib/renterd/renterd.yml
 ```
+{% endtab %}
+
+{% tab title="Zen Test Net" %}
+```console
+sudo nano /var/lib/renterd_zen/renterd.yml
+```
+{% endtab %}
+{% endtabs %}
 
-Now, modify the file to add your wallet seed and API password. The recovery phrase is the 12-word phrase you generated in the previous step. Type it carefully, with one space between each word, or copy it from the previous step. The password is used to unlock the `renterd` UI; it should be something secure and easy to remember.
+Now, modify the file to add your wallet seed and API password. The recovery phrase is the 12-word phrase you generated in the previous step. Type it carefully, with one space between each word, or copy it from the previous step. The password is used to unlock the `renterd` web UI; it should be something secure and easy to remember.
 
-{% hint style="info" %}
-If you are planning to use `renterd`'s built-in S3 API, make sure you enable S3 in your `renterd.yml` file.
+{% hint style="warning" %}
+`your_access_key` can be anywhere from 16 to 128 characters long, and `your_private_key` must be exactly 40 characters long.
 {% endhint %}
 
 {% tabs %}
```

Comment:
There's no need to separate these. Remove the address portion and they're the same.

Reply:
My thinking was it would be good to separate them in case the user is already running a version of `renterd` and would like to run a Testnet instance alongside it. Wouldn't having them on the same port cause issues?

Reply:
Never mind, I see what you're saying now.